src/app/inventory/store/d2-item-factory.ts

```diff
@@ -554,10 +559,24 @@ export function makeItem(
   if (createdItem.objectives) {
     const length = createdItem.objectives.length;
     if (length > 0) {
+      const checkTrialsPassage = isTrialsPassage(createdItem, defs);
```

Comment:
Move this down to where it's used, we don't need to check it unless `objective.completionValue` is truthy